quiche/src/lib.rs

```diff
@@ -3804,7 +3804,7 @@ impl<F: BufFactory> Connection<F> {
 
         // Update max datagram size to allow path MTU discovery probe to be sent.
         if send_path.pmtud.get_should_probe() {
-            let size = if self.handshake_confirmed || self.handshake_done_sent {
+            let size = if self.handshake_confirmed || self.handshake_confirmed {
```

Comment:
This should be:

`let size = if self.handshake_confirmed || self.handshake_completed {`

Reply:
Yeah I fixed